code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -153,6 +152,13 @@
 	else
 		. += "<span class='notice'>It is spent.</span>"
 
+/obj/item/reagent_containers/hypospray/autoinjector/epinephrine
+	name = "emergency autoinjector"
+	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge."
+	icon_state = "autoinjector"
```

Comment:
If the name/description/icon are meant to be the same as its parent type's, it'll just inherit them so you don't need to restate it
```suggestion
```

Reply:
Good one, missed this!